### PR TITLE
chore: replace exec-maven-plugin with gradle-maven-plugin

### PR DESCRIPTION
**Pull request by manolo**
This should fix problems when running the build in docker containers in windows.

### CODE REVIEW

File: flow-plugins/flow-gradle-plugin/pom.xml [hunk 1]
```diff
@@ -55,34 +55,40 @@
...
+                        <arg>functionalTest</arg>
+                        <arg>-S</arg>
+                    </args>
+                </configuration>
                 <executions>
                     <execution>
-                        <id>gradle</id>
                         <phase>prepare-package</phase>
-                        <configuration>
-                            <executable>${gradle.executable}</executable>
```
Comment (ZheSun88) [inline flow-plugins/flow-gradle-plugin/pom.xml:69]: this property can be removed

**Comment by ZheSun88** [comment]
this fix runs good on windows..(but it failed here with the flow validation/it-tests(6,3) )

image

File: .github/workflows/validation.yml [hunk 2]
```diff
@@ -31,6 +31,8 @@ jobs:
             | sort > it.modules &
           P3=$!
           wait $P1 $P2 $P3
+      # - name: Sleep
+      #   run: echo Sleeping && sleep 300
```
Comment (ZheSun88) [inline .github/workflows/validation.yml:35]: maybe remove this?

**Review by ZheSun88 — APPROVED** [review]

File: flow-plugins/flow-gradle-plugin/pom.xml (hunk 1 above)
Comment (manolo) [reply flow-plugins/flow-gradle-plugin/pom.xml:69]: done

**Review by ZheSun88 — APPROVED** [review]

**Pull request merged**